qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1520A.py

```diff
@@ -161,6 +517,215 @@ def abort(self):
+
+            adc_coef: the number of averaging samples or the
+                averaging time.
+
+            imp_model: specifies the units of the parameter measured by the
+                MFCMU.
+
+            ranging_mode: Auto range or Fixed range
+
+            fixed_range_val: Integer 0 or more. Available measurement ranges
+                depend on the output signal frequency.
+
+            hold_delay: Hold time (in seconds) that is the wait time after
+                starting measurement and before starting delay time for the
+                first step 0 to 655.35, with 10 ms resolution.
+
+            delay: Delay time (in seconds) that is the wait time after
+                starting to force a step output and before starting a step
+                measurement.
+
+            step_delay: Step delay time (in seconds) that is the wait time
+                after starting a step measurement and before starting to
+                force the next step output. 0 to 1, with 0.1 ms resolution.
+                If  step_delay is shorter than the measurement time,
+                the B1500 waits until the measurement completes, then forces
+                the next step output.
+
+            trigger_delay: Step source trigger delay time (in seconds) that
+                is the wait time after completing a step output  setup and
+                before sending a step output setup  completion trigger. 0 to
+                delay, with 0.1 ms resolution.
+
+            measure_delay: Step measurement trigger delay time (in
+                seconds) that is the wait time after receiving a start step
+                measurement trigger and before starting a step measurement.
+                0 to 65.535, with 0.1 ms resolution.
+
+            abort_enabled: Boolean, enables or disables the automatic abort
+                function for the CV sweep measurement.
+
+            sweep_mode: Linear sweep, log sweep, linear 2 way sweep or
+                log 2 way sweep
+
+            volt_monitor: Boolean
+
+        """
+
+        self.root_instrument.enable_channels(self.channels)
+        self.adc_mode(adc_mode)
+        self.adc_coef(adc_coef)
+        self.frequency(freq)
+        self.voltage_ac(ac_rms)
+        self.cv_sweep.sweep_auto_abort(abort_enabled)
+        self.cv_sweep.post_sweep_voltage_cond(post_sweep_voltage_cond)
+        self.cv_sweep.hold(hold_delay)
+        self.cv_sweep.delay(delay)
+        self.cv_sweep.step_delay(step_delay)
+        self.cv_sweep.trigger_delay(trigger_delay)
+        self.cv_sweep.measure_delay(measure_delay)
+        self.sweep_mode(sweep_mode)
+        self.sweep_start(v_start)
+        self.sweep_end(v_end)
+        self._sweep_steps = n_steps
+        self.sweep_steps(self._sweep_steps)
+        self.measurement_mode(constants.MM.Mode.CV_DC_SWEEP)
+        self.impedance_model(imp_model)
+        self.ac_dc_volt_monitor(volt_monitor)
```

Comment:
LET"S use THIS as TRUE!!! in order not to calculate voltage ourselves.

Reply:
Done. I also had a discussion with John offline and he was happy to know that making this 'True' gives us exact voltages. He is sure that this will not have any significant increase in the sweep time. He also agreed to make this TRUE.